autogpt_platform/backend/backend/integrations/webhooks/slantthreed.py

```diff
@@ -0,0 +1,108 @@
+from typing import ClassVar, Dict, List, Tuple
```

Comment:
Pls use `dict`, `list`, `tuple` instead of `Dict`, `List`, `Tuple`

Reply:
done